src/ScottPlot/Tools.cs

```diff
@@ -322,6 +322,15 @@ public static string GetOsName(bool details = true)
 
             return name;
         }
+      
+        public static int SimpleHash(double[] input)
+        {
+            byte[] bytes = input.SelectMany(n => { return BitConverter.GetBytes(n); }).ToArray();
+            int hash = 0;
+            foreach (byte b in bytes)
+                hash = (hash * 31) ^ b;
```

Comment:
Double.GetHashCode() might be more convenient.

Of course C# has plenty of other hashing algorithms if you feel like it:

![image](https://user-images.githubusercontent.com/8635304/85185445-2b7e1600-b251-11ea-8240-ef017bad5106.png)

Reply:
Good points! However, I added this to help that an array created by a C# program was identical to one created by a Python program, so this seemed like a simple solution that didn't depend on an an external library or language-specific hashing mechanism.

As an aside, I'm aware that having a "tools", "utils", or "misc" class is a code smell 😅 

Note the MD5 method is used to create Bitmap hashes:
https://github.com/swharden/ScottPlot/blob/85cc7b022e2d26db3ee08933d9614013766f4485/src/ScottPlot/Tools.cs#L52-L61